apps/prairielearn/src/lib/question-render.sql

```diff
@@ -57,6 +57,7 @@ SELECT
   s.duration,
   s.gradable,
   s.graded_at,
+  s.grading_method,
```

Comment:
This column was never returned, even though this includes every `submissions` column EXCEPT for jsonb columns.

Reply:
The grading_method column in submissions is being deprecated. See #11514.

Reply:
Added #12408 to drop the column. That said, assuming you're planning an automated check, there should probably be a plan to handle the multi-stage process of handling a deleted column, since you'd need to remove it from queries and the schema in one PR, while deleting it from the DB in a follow-up PR.